internal/provider/resource_network_pool_test.go

```diff
@@ -7,17 +7,16 @@ import (
 	"fmt"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
-	"github.com/vmware/terraform-provider-vcf/internal/api_client"
 	"github.com/vmware/terraform-provider-vcf/internal/constants"
 	"log"
 	"testing"
 )
 
 func TestAccResourceVcfNetworkPool(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:          func() { testAccPreCheck(t) },
-		ProviderFactories: providerFactories,
-		CheckDestroy:      testCheckVcfNetworkPoolDestroy,
+		PreCheck:                 func() { testAccPreCheck(t) },
```

Comment:
update copyright year. Sorry for placing this comment here, but Github doesn't allow comments on unchanged lines. Applicable for whole commit.

Reply:
fixed